README.md

```diff
@@ -31,7 +31,12 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
     `git clone https://github.com/NuGet/NuGet.Client`
 
-2. Start PowerShell. CD into the cloned repository directory.
+2. Start Develooper Command Prompt for VS. Then, start PowerShell and navigate into the cloned repository directory:
```

Comment:
It can function without a developer command prompt. 

If it doesn't, we need to fix it so it does.

Reply:
Should we do this in another PR?

Reply:
Yes, if it doesn't work we should create an issue that it doesn't and fix it.

Reply:
Ok. I will revert the change and create the corresponding issue.

Reply:
Created the issue: https://github.com/NuGet/Home/issues/7914